brightwind/analyse/analyse.py

```diff
@@ -2065,3 +2066,66 @@ def calc_air_density(temperature, pressure, elevation_ref=None, elevation_site=N
         raise TypeError('elevation_ref should be a number')
     else:
         return ref_air_density
+
+def extrapolate_pressure(reference_height_m, target_height_m, reference_pressure_hPa_mbar, reference_temp_degC, print_details = False):
+    """
+    Calculates pressure at target height (target_height_m) using reference pressure (reference_pressure_hPa_mbar) and reference temperature (reference_temp_degC) values (reference values valid at height of reference_height_m).
+
+    Calculation based on ISO:2533-1975 Standard Atmosphere (https://www.iso.org/obp/ui/#iso:std:iso:2533:en) as suggested by IEC standard (61400-12-1).
+    
+    :param reference_height_m:                Height (in metres) to extrapolate pressure to / height of output pressure
+    :type reference_height_m:                 Float or pandas.Series
+    :param target_height_m:                   Height (in metres) at which reference pressure (reference_pressure_hPa_mbar) and reference temperature (reference_temp_degC) are valid
+    :type target_height_m                     Float or pandas.Series
+    :param reference_pressure_hPa_mbar:       Reference pressure value(s) in mbar or hPa (1mbar = 1hPa = 100Pa)
+    :type extrapolate_from_pressure_hPa_mbar:       Float or pandas.Series
+    :param reference_temp_degC:               Reference temperature value(s) in degrees celcius
+    :type reference_temp_degC:                Float or pandas.Series
+    :param print_details:                     If True, print details of the calculation and output
+    :type print_details:                      Boolean (default False)
+    :return:                                  Pressure at specified height of target_height_m in Pa
+    :rtype:                                   Float or pandas.Series depending on type(reference_pressure_hPa_mbar)/type(reference_temp_degC) inputs
+
+        **Example usage**
+    ::
+
+    import brightwind as bw
+
+    bw.extrapolate_pressure(reference_height_m=10, target_height_m=200, reference_pressure_hPa_mbar=1000, reference_temp_degC=12)
```

Comment:
before each reported examples you should add a short description of what you want to get with your example e.g. 
`# extrapolate air pressure giving as input reference variables as a float and not print out details of the calculation `